integration-tests/micrometer-mp-metrics/src/main/java/io/quarkus/it/micrometer/mpmetrics/PrimeResource.java

```diff
@@ -70,6 +70,8 @@ String checkPrime(long number) {
     }
 
     @Gauge(name = "highestPrimeNumberSoFar", unit = MetricUnits.NONE, description = "Highest prime number so far.")
+    @GET
+    @Path("/blabla") // make this a REST method just to verify that a gauge will still work on it
```

Comment:
Do we have to add something to the test that will use it?

Reply:
No, the test already checks the value of that metric for some unrelated purposes, which is enough